test/test_nn.py

```diff
@@ -8326,6 +8326,51 @@ def test_functional_grad_conv(self):
             torch.nn.grad._grad_input_padding(torch.rand(1, 2, 3), [1, 2, 5], (1,), (0,), (3,))
         self.assertEqual(len(w), 1)
 
+    def test_unflatten(self):
+        tensor_input = torch.randn(32, 49152)
```

Comment:
nit: could you make the input Tensor smaller here? Just to make sure we don't slow down the tests for no reasons.